content/livesync/connector/index.textile

```diff
@@ -199,15 +218,17 @@ ADBC_NODES_TABLE_AUTO_CREATE=true
 
 Railway will restart the @adbc-test@ service with the newly applied configuration.
 
-With a PostgreSQL database and Database Connector instance running on Railway, the next step is to write a record to the outbox table and confirm it has been published over Ably:
+h3(#varify-railway). Varify connection using Railway
 
-Open a terminal and subscribe to a "channel":/channels named @foo@ using "Server-Sent Events":/protocols/sse
+Varify your connection to the Ably network now you have a PostgreSQL database and Database Connector instance running. The following examples show how to write a record to the outbox table and confirm its publication. 
```

Comment:
```suggestion
Verify your connection to the Ably network now you have a PostgreSQL database and Database Connector instance running. The following examples show how to write a record to the outbox table and confirm its publication. 
```

Reply:
Thanks, Greg. I was convinced it was "varify" lol